scripts/install.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+function phala_scripts_install_aptdependencies() {
+  if [ "$1" == "uninstall" ];then
+    shift
+    phala_scripts_log info "Uninstall Apt dependencies" cut
+    apt autoremove -y $*
+    return 0
+  fi
+
+  _default_soft=$*
+  phala_scripts_log info "Apt update" cut
+  apt update
+  if [ $? -ne 0 ]; then
+    phala_scripts_log error "Apt update failed."
+  fi
+  phala_scripts_log info "Installing Apt dependencies" cut
+  apt install -y ${_default_soft[@]}
+}
+
+function phala_scripts_install_otherdependencies(){
+  if [ "$1" == "uninstall" ];then
+    shift
+    phala_scripts_log info "Uninstall other dependencies" cut
+    for _package in $*;do
+      if ! type $_package > /dev/null 2>&1;then
+        :
+      else
+        case $_package in
+          docker)
+            apt autoremove -y docker-ce docker-ce-cli
+            find /etc/apt/sources.list.d -type f -name docker.list* -exec rm -f {} \;
+          ;;
+          docker-compose)
+            [ -f /usr/local/bin/docker-compose ] && rm -rf /usr/local/bin/docker-compose
+          ;;
+          node)
+            apt autoremove -y nodejs
+            find /etc/apt/sources.list.d -type f -name nodesource.list* -exec rm -f {} \;
+          ;;
+          *)
+            apt autoremove -y $_package
+          ;;
+        esac
+      fi
+    done
+    return 0
+  fi
+
+  _other_soft=$*
+  phala_scripts_log info "Installing other dependencies" cut
+  for _package in ${_other_soft};do
+    if ! type $_package >/dev/null 2>&1;then
+      case $_package in
+        docker-compose)
+          if [ ! -f /usr/local/bin/docker-compose ];then
+            if [ "${PHALA_LANG}" == "CN" ];then
+              curl -L "https://get.daocloud.io/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o ${phala_scripts_tmp_dir}/docker-compose && \
```

Comment:
Hope we can extract URL like this to constant, then we shall easy to maintain them (e.g. upgrade for security reason)